politeiawww/cmd/piwww/testrun.go

```diff
@@ -45,9 +45,11 @@ func login(email, password string) error {
 	return lc.Execute(nil)
 }
 
-// newProposal returns a NewProposal object contains randomly generated
-// markdown text and a signature from the logged in user.
-func newProposal() (*v1.NewProposal, error) {
+// newProposal returns a NewProposal object contains randonly generated
+// markdown text and a signature from the logged in user, if given RFP
+// bool is true it creates an RFP.
+// If given linkto it creates a RFP submission.
+func newProposal(RFP bool, linkto string) (*v1.NewProposal, error) {
```

Comment:
When you have a bool as a function argument, I like to create a more descriptive wrapper for it to improve the readability of your code.

For example, without a wrapper:
```
np, err := newProposal(true, tokenA)

np, err := newProposal(false, tokenA)
```

With a wrapper:
```
np, err := newProposalRFP(tokenA)

np, err := newProposalStandard(tokenA)
```
where `newProposalRFP` and `newProposalStandard` are just wrappers:
```
func newProposalRFP(linkto string) (*v1.NewProposal, error) {
  return newProposal(true, linkto)
}
```

If you're reading the code which one is easier to read and reason about?

Reply:
Make total sense, will do.